llvm/lib/Target/NVPTX/NVPTXInstrInfo.td

```diff
@@ -859,12 +857,12 @@ def MULWIDEU32Imm :
 def MULWIDEU32Imm32 :
```

Comment:
After your removal of the below chunk some of these instructions look to be dead.